[2021-06-25 09:17:40,781] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-02-27T00:00:00+00:00 [queued]>
[2021-06-25 09:17:41,044] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-02-27T00:00:00+00:00 [queued]>
[2021-06-25 09:17:41,045] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:17:41,045] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-25 09:17:41,060] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 09:17:41,233] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-02-27T00:00:00+00:00
[2021-06-25 09:17:41,301] {standard_task_runner.py:52} INFO - Started process 5318 to run task
[2021-06-25 09:17:41,396] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-02-27T00:00:00+00:00', '--job-id', '219', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp8yv47we0', '--error-file', '/tmp/tmpmivi2lz0']
[2021-06-25 09:17:41,400] {standard_task_runner.py:77} INFO - Job 219: Subtask task_1
[2021-06-25 09:17:42,132] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-02-27T00:00:00+00:00 [running]> on host airflowvm
[2021-06-25 09:17:43,444] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-02-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-27T00:00:00+00:00
[2021-06-25 09:17:43,446] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-06-25 09:17:43,509] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-06-25 09:17:43,645] {subprocess.py:75} INFO - Output:
[2021-06-25 09:17:46,744] {subprocess.py:83} INFO - Command exited with return code 0
[2021-06-25 09:17:47,172] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200227T000000, start_date=20210625T091740, end_date=20210625T091747
[2021-06-25 09:17:49,723] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-25 09:17:49,882] {local_task_job.py:151} INFO - Task exited with return code 0
